cli/src/main/java/com/liferay/blade/cli/command/DeployArgs.java

```diff
@@ -25,10 +25,22 @@
 @Parameters(commandDescription = "Builds and deploys bundles to the Liferay module framework.", commandNames = "deploy")
 public class DeployArgs extends BaseArgs {
 
+	public boolean isLegacy() {
+		return _legacy;
+	}
+
 	public boolean isWatch() {
 		return _watch;
 	}
 
+	
+
+	@Parameter(
```

Comment:
Lets instead create a branch new 'extension' that is just restores the old custom command.  Call it 'deployRemote' or 'remoteDeploy' or another idea?